### ISSUE TITLE
Outstanding Items for the Segmented Gather Example Presentations

### ISSUE THREAD

**Issue by edwardsarah** [hackforla/guides#112]
### Overview
A list of action items for the smaller gather example presentations so they can be completed. 

### Action Items
- [ ] Standardize formatting across presentations
- [ ] Update table of contents for each slide deck & relink correct slides
- [ ] Title slides for each smaller presentation
- [ ] Ending slides (Q&A, Contributors - basically 83 and beyond in original presentation)
- [ ] Update section end messages (congrats! you've successfully done x) 

### Resources/Instructions
Original Gather Examples Presentation:
[GP: How to Gather Examples and Interview Teams](https://docs.google.com/presentation/d/1gkPpgnOPv7lBckoLic3HAA7z_IkeTnCw9ImQp9hkujM/edit?usp=sharing)
Smaller Presentations:
[What is a Guide & Picking Your First Guide](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit?usp=sharing)
[Gathering Examples: Github [Wiki, Issues, Project Board, Resources Provided]](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit?usp=sharing)
[Gathering Examples: Slack](https://docs.google.com/presentation/d/1-Kg7yLRXsdGo4Ln8L-WiunOOO3Uc7TuXvH5afcp3AKQ/edit?usp=sharing)
[How to Connect w/ Other Teams, Moving the Conversation Forward, How to Interview Practitioners](https://docs.google.com/presentation/d/15WxdSc_YFVLdytCZ-SxZaWPt8y6-OQTxJb8dcppZHVE/edit?usp=sharing)
[Making a Special Issue & Last Check w/ Org Rep](https://docs.google.com/presentation/d/1egg1WUxIt9lz4mFXiR7OeKb6dTz-5HCxi5Ai8uEL4Ps/edit?usp=sharing)

**Comment by edwardsarah** [comment]
Moving forward from: https://github.com/hackforla/guides/issues/105#issue-1209810402

**Comment by edwardsarah** [comment]
Do we want to make slack method 5 and resources provided method 4? The way the presentations are currently split up, it goes 1, 2, 3, 5 in the first presentation, and then method 4 in its own presentation.

**Comment by SheIsBukki** [comment]
> Do we want to make slack method 5 and resources provided method 4? The way the presentations are currently split up, it goes 1, 2, 3, 5 in the first presentation, and then method 4 in its own presentation.

Yes, I have edited them accordingly

**Comment by edwardsarah** [comment]
switched the contents of the table of contents and the section contents in presentation 3 based on discussion we had last week. otherwise, I don't think there's currently any other edits we need to make to the presentation.

**Comment by SheIsBukki** [comment]
> switched the contents of the table of contents and the section contents in presentation 3 based on discussion we had last week. otherwise, I don't think there's currently any other edits we need to make to the presentation.

Awesome!

**Comment by ExperimentsInHonesty** [comment]
There have been some changes made so the table on contents needs to be rechecked

**Comment by ExperimentsInHonesty** [comment]
I used this clipart of a monitor on one of the slides https://hdclipartall.com/monitor-clipart-monitor-clipart-850_706-156447.html

**Comment by edwardsarah** [comment]
The colored boxes in Part 5 don't seem to conform to the standards of colored boxes in the other presentations, can we discuss this on Monday?

**Comment by edwardsarah** [comment]
'Benefits for you as X' boxes
- Part 1 has one (sort of) on [slide 8](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g117fac9d1a0_0_15)
- Part 3 has one on [slide 5](https://docs.google.com/presentation/d/1-Kg7yLRXsdGo4Ln8L-WiunOOO3Uc7TuXvH5afcp3AKQ/edit#slide=id.g13bf0e9d4d2_0_77)
- Parts 4 & 5 have none

Part 2 has 'benefits for x' boxes on each method intro slide, and a slide dedicated to the overall [benefits of gathering examples](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g150e9519a04_1_360)
- [Wiki](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.gfcb8642ab3_2_124)
- [Issues](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g13bf0e9d4d2_0_61)
- [Project Board Card](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g13bf0e9d4d2_0_69)
- [Resources Provided](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g14785c83767_0_187)

**Comment by edwardsarah** [comment]
Change all boxes to what to observe w dashed border

**Comment by edwardsarah** [comment]
@SheIsBukki did you have a chance to review the slide decks yet? if you leave a comment on parts where we need steps and don't have them yet I can help fix those slides for Friday

**Comment by edwardsarah** [comment]
Part 1 has been checked completely and does not need another review.

**Comment by edwardsarah** [comment]
Speaker Notes in Presentations:
none in part 1 or part 3, except the link to Bonnie's linkedin page that shows up on every Contributors page

**Part 2:** 
- slide 19 (an exercise)
- slide 25 (link to an example issue used)
- slide 26 (an exercise)

**Part 4:** 
- slide 5 (template text)
- slide 6 (links to example issues used)
- slide 7 (template text)
- slide 8 (links to example issued used)
- slide 9 (template text)
- slide 11 (link to flowchart)
- slide 14 (template text & example responses)
- slide 17 (template text)

**Part 5:** 
- slide 6 (additional example)
- slide 10 (template text)
- slide 11 (template text)
- slide 12 (template text)

**Comment by edwardsarah** [comment]
![open_in_new_FILL0_wght400_GRAD0_opsz48](https://user-images.githubusercontent.com/64277558/202961348-43e91824-6ce7-43c7-a23a-a6095f90a1dd.png)
used in part 3

from [google material symbols](https://fonts.google.com/icons)

**Comment by edwardsarah** [comment]
Updates ahead of 1/6 meeting:
- Glossary words identified on slides and added to footer. 
- Section on slack files has been updated with screenshots for an example shared in multiple slack channels.

**Comment by ExperimentsInHonesty** [comment]
@seannwvong is goign to start with [How to Make a Guide: Overview](https://docs.google.com/presentation/d/1JTf5oZ70DwvAEd5_Tco-spgvmdsiYVu2kgd9tAyIuws/edit)

**Comment by edwardsarah** [comment]
Titles of the presentations (and references to them within the presentations & the overview deck) have been edited to reflect the decision we made last Friday to separate Part 1 (now 'What is a Guide') from the rest of the Gather Examples decks.

**Comment by seannwvong** [comment]
# [What is a Guide & Picking Your First Guide](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit) 
review and comments

- [ ] [Slide 1:Cover](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g14b05c93f55_0_72)
  - Added hyphen to change to "Self-Assigning"
  - The "[How to Make a Guide: Overview](https://docs.google.com/presentation/d/1JTf5oZ70DwvAEd5_Tco-spgvmdsiYVu2kgd9tAyIuws/edit)" cover slide is red. I thought it was black when I reviewed. Should the cover slides of other decks be red also for consistency?
- [ ] [Slide 2: Purpose](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g124ce1f2c2b_1_63)
  - Should "Guide team be "Guides team"?
  - In the "[How to Make a Guide: Overview](https://docs.google.com/presentation/d/1JTf5oZ70DwvAEd5_Tco-spgvmdsiYVu2kgd9tAyIuws/edit#slide=id.g18be17613f0_0_122)" Purpose slide, this line has a wiki icon at the end, but there's no icon here. We should be consistent.
- [ ] [Slide 6: How to See Guides in Process](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g127639ce232_0_1)
  - Will there be a wiki where others can see the guides that are ready? It's not immediately clear to me on the Kanban board which guides are "done done".
  - Side note: the Kanban board definition is "visual system used to manage and keep track of work as it moves through a process." The Guides Tracker Kanban columns appear to morph into a grouping area of different statuses when we scroll to the right; doesn't exactly illustrate a process. Happy to learn more about different nuances and how/if we can optimize this.
- [ ] [Slide 7: Guide Types & Examples](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g12535235506_2_6)
  - The intent of this slide at this point in the deck to highlight 2 types of guides is sound but the examples are not Guide examples. They are all issues and checklists on how to create the guide. As a reader I would have to dig further to find the actual guide. Perhaps it could be listed as _Managing your project's Slack channel ([guide](url) | [issue](url))_ if there are different links for each guide.
- [ ] [Slide 10: How To Find the right Complexity Level](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g133aaa1b952_0_69)
  -  At this point of the deck I haven't actually seen a Guide so I'm not sure where I would see this label.
  - If the guide doesn't have the complexity labels, should users read that as "this is unscoped" and hang on before starting to create the guides?
  - Who does the scoping and labelling for each guide?
- [ ] [Slide 11: Find a New Guide to Work on](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g117fac9d1a0_0_50)
  - I think it might be useful to have screenshots with issues that have the Complexity labels to emphasize the previous slide.
  - Only Ops have scoped the guides, but they are in point format, not Complexity labels. There should be some consistency in what we're trying to communicate complexity.
  - Glossary for Prioritized Backlog should appear here as it's the first appearance.
  - What's the difference between Guides here and Guides in the Guides Tracker? Do they display the same guides i.e. the Guides Tracker is a roll up of all the Guides in the CoP?
- [ ] [Slide 12: How to Select a New Guide](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g117fac9d1a0_0_26)
  - I think the crosses and ticks are examples that can be verbally presented but it isn't clear how as a reader I should understand why the middle one is a tick for everyone.
  - Prioritized Backlog first appeared in Slide 11 but I'm fine with displaying glossary here.
- [ ] [Slide 16: Congrats](https://docs.google.com/presentation/d/1WvHkRja7McjraE-QIjMSPluEOwBbLhFletlCYnzGLb0/edit#slide=id.g133aaa1b952_0_9)
  - At this point of communicating "how to create guides", would "Overview" have been presented already? Listing Overview here as a next step and after Gathering Examples seems confusing. Also not immediately clear what "Overview" is overview for or why. May I suggest another bullet point to describe if readers want to find out more about the overview of creating guide, see the Overview(link) deck.

**Comment by seannwvong** [comment]
# [Part 1: Gathering Examples With Github [Wiki, Issues, Project Board, Resources Provided]](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit) 
review and comments

- [ ] [Slide 5: Benefits from Gathering & Cataloging Examples](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g150e9519a04_1_360)
  - [ ] Author definition in glossary is vague. "A person who writes the guide. Other people who work on a guide but are not drafting the majority of it are contributors. The first person to make a guide document is listed first under contributors at the bottom of the guide." 1st and 2nd line distinguishes authors from contributors but the subsequent line groups authors and contributors under the contributors section.
- [ ] [Slide 9: How to Find and Copy the Projects List and their Repositories](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g1530624dcad_9_15)
  - [ ] Updated bullet list of projects into a checklist so that it becomes a checklist when copied. 
- [ ] [Slide 11: How will the Projects List be Used?](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g1530624dcad_9_31)
  - [ ] It makes sense for this "meeting notes" list to have bullets as we're always having meetings, technically  the list will never technically be complete so making use of a checklist less useful. But, should a checklist still be used where research scope is more fixed?
- [ ] [Slide 13: Methods for Gathering Examples from Teams](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g150e9519a04_1_354)
  - [ ] Took the liberty to link to the respective sections
  - [ ] Struggling with this verb. The method is the same - checking all these areas. Should this be "sections in GitHub" where we should search for?
- [ ] [Slide 14: Explore a project to see if a team is using the method](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g1530624dcad_8_19)
  - [ ] Same comment as previous - should this be areas where authors should check instead.
- [ ] [Slide 19: Method 1: Exercise](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g11b172cbaeb_1_0)
  - [ ] rephrased from "when finding" to "to find"
- [ ] [Slide 28: Method 3: Using Project Board Cards](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g13bf0e9d4d2_0_69)
  - [ ] Would/Should we (the ones crafting the guide) view cards? Is this section still relevant? We should explain we should still check out the classic project boards because there's where the historical data would/might be. The comms style here should be positive i.e. what we can see and why, not just a negative i.e. beta boards don't allow cards.
- [ ] [Slide 29: Method 3: What is a Project Board Card Link?](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g183121c13a4_3_74)
  - [ ] Previous slide calls out "project board card" and this slide talks about "project board card link". They appear synonymous but are not exactly the same. Slide 34 references copying a link to the relevant cards. For this slide we should remove the word "link".
- [ ] [Slide 41: Next Steps](https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit#slide=id.g2110735e355_0_4)
  - [ ] Added new slide based on last week's discussion on creating "journey" overview. Probably need UX designer to improve it.

**Comment by edwardsarah** [comment]
## Non-presentation Questions
- What date is the CoP leads meeting next month & what time are these meetings generally at? 

## New Slides / Slide Updates
### Overview
- Not sure if we got to it last week, but [slides 6-9] are new. 6 & 7 are reworked to include the additional steps we talked about, and 8 we discussed adding two weeks ago. I still have to make sure all of the WIP labels have been added to the correct decks. 
- [Leadership review slide](https://docs.google.com/presentation/d/1JTf5oZ70DwvAEd5_Tco-spgvmdsiYVu2kgd9tAyIuws/edit#slide=id.g22dfd0c524c_0_10) has been edited to mention reaching out to your CoP leads when you're ready for leadership review. There's a lot going on in this slide, so maybe we should split it into two? 

### How to Use Github Search
- Basically all of the slides in [What You Can Search](https://docs.google.com/presentation/d/1M2f2Q9vefNAzQ4lvcw13x-7McPbrb8NZIVFgJUrefzE/edit#slide=id.g2f8203f177623846_19) have been reworked
### How to Ask for Peer Review
- [Comment Etiquette Slide Added](https://docs.google.com/presentation/d/1srOj8VsNfIdqQK5fH2hxTrSA7JJ94APnoYoghAa09pM/edit#slide=id.g1e24f25cb4f_0_6) 
Any suggestions on better examples of bad & good feedback are welcome

## Blockers/Questions
- In How to Use Github Search: does the flow of slides 6-9 make sense? Is there a better way to organize this information? 
- Is the deck on Github global search meant to be part of Part 1 on searching Github, does it function like Part 1.5, or are we adding it as a new part between parts 1 & 2? Or is it more of an appendix? 
- Is the [search syntax slide](https://docs.google.com/presentation/d/1M2f2Q9vefNAzQ4lvcw13x-7McPbrb8NZIVFgJUrefzE/edit#slide=id.g23b00743ff1_0_132) too busy? 
- Editing some steps in how to ask for peer review are on hold until I understand what's standard across CoPs. This will just require a little more looking around on my part. Not every CoP uses the same columns on their guides board or the exact same labels for their guides. 

## My Next Steps
- Fixing the exercise in part 3 that we discussed last week! 
- How to Ask for Peer Review slide deck needs screenshots & template text. I will look around in issues/CoP slacks to see if there are any examples I can pull from from there, otherwise I might as UI/UX if they have any suggestions
- How to Peer Review a Guide needs a few slides on wrapping up the peer review process & formatting changes like fixing the appendix & 'map' slide at the end
- Overview needs to be updated with Github global search deck
